src/WebJobs.Extensions.DurableTask/DurableTaskJobHostConfigurationExtensions.cs

```diff
@@ -107,6 +109,33 @@ public static IWebJobsBuilder AddDurableTask(this IWebJobsBuilder builder)
             return builder;
         }
 
+#if FUNCTIONS_V3_OR_GREATER
+        /// <summary>
+        /// Adds the <see cref="IScaleMonitor"/> and <see cref="ITargetScaler"/> providers for the Durable Triggers.
+        /// </summary>
+        /// <param name="builder">The <see cref="IWebJobsBuilder"/> to configure.</param>
+        /// <returns>Returns the provided <see cref="IWebJobsBuilder"/>.</returns>
+        public static IWebJobsBuilder AddDurableScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
```

Comment:
Please make the method as internal. We decided to use reflection to call the extensions methods.

Reply:
Sounds good, it's been addressed here: https://github.com/Azure/azure-functions-durable-extension/pull/2462/commits/3800635bc41957c9ec60880755dc5b4914d90868

Reply:
@alrod: does this mean that we need to change how ScV3 calls into our extension method? If so, can you please link @bachuv to an example of how to change this in the Sc codebase? thanks!

Reply:
I will change calling the method once we have the nuget package and update the reference in SC v3.x.